src/commandline_frame.ts

```diff
@@ -47,6 +47,21 @@ export function changecompletions(newcompletions: string) {
     completions.innerHTML = newcompletions
 }
 
+export function setclipboard(content: string) {
+    clInput.value = content
+    clInput.select()
+    document.execCommand("Copy")
+    // todo: Maybe we can consider to using some logger and show it with status bar in the future
+    console.log('set clipboard:', content)
+}
+
+export function gotourl() {
+    clInput.focus()
+    document.execCommand("Paste")
+    const url = clInput.value
+    console.log('goto url:', url);
```

Comment:
Not done yet, I think we need to notify the content document to do this jump.

Reply:
I think it makes more sense to have a dedicated textarea for copying and pasting and an interface here that just offers copying and pasting.

The paste command can just return a string that the caller can do something with.

Then excmds.ts:clipboard becomes something like:

```ts
export async function clipboard(excmd = "open"){
    switch (excmd) {
        case 'yank':
            const atab = await activeTab()
            await message("commandline_frame", "setclipboard", [atab.url])
            break
        case 'open':
	    open(await message("commandline_frame", "paste"))
            break
        default:
            // todo: maybe we should have some common error and error handler
            throw new Error(`[clipboard] unknown excmd: ${excmd}`)
    }
    hidecmdline()	
}
```

Maybe the functions should be called `getclipboard` rather than paste. Your discretion.

I'm not sure if excmds.ts uses the better messaging system yet. If not, I'll fix it after this review.

Reply:
I want to do this at first, just cause handler in `commandline_frame` does not deal with sendResponse, I can do this.